tests/acceptance/features/coreApiWebdavOperations/search.feature

```diff
@@ -186,23 +186,23 @@ Feature: Search
-      | a:getetag             |
-      | a:getcontenttype      |
+      | d:getlastmodified     |
+      | d:getetag             |
+      | d:getcontenttype      |
       | oc:size               |
       | oc:owner-id           |
       | oc:owner-display-name |
     Then the HTTP status code should be "207"
     And file "/upload.txt" in the search result of user "Alice" should contain these properties:
-      | name                                       | value                                                                                             |
-      | {http://owncloud.org/ns}fileid             | \d*                                                                                               |
-      | {http://owncloud.org/ns}permissions        | ^(RDNVW\|RMDNVW)$                                                                                 |
-      | {DAV:}getlastmodified                      | ^[MTWFS][uedhfriatno]{2},\s(\d){2}\s[JFMAJSOND][anebrpyulgctov]{2}\s\d{4}\s\d{2}:\d{2}:\d{2} GMT$ |
-      | {DAV:}getetag                              | ^\"[a-f0-9:\.]{1,32}\"$                                                                           |
-      | {DAV:}getcontenttype                       | text\/plain                                                                                       |
-      | {http://owncloud.org/ns}size               | 15                                                                                                |
-      | {http://owncloud.org/ns}owner-id           | %username%                                                                                        |
-      | {http://owncloud.org/ns}owner-display-name | %displayname%                                                                                     |
+      | name                  | value                                                                                             |
+      | oc:fileid             | \d*                                                                                               |
+      | oc:permissions        | ^(RDNVW\|RMDNVW)$                                                                                 |
+      | d:getlastmodified     | ^[MTWFS][uedhfriatno]{2},\s(\d){2}\s[JFMAJSOND][anebrpyulgctov]{2}\s\d{4}\s\d{2}:\d{2}:\d{2} GMT$ |
+      | d:getetag             | ^\"[a-f0-9:\.]{1,32}\"$                                                                           |
+      | d:getcontenttype      | text\/plain                                                                                       |
+      | oc:size               | 15                                                                                                |
+      | oc:owner-id           | %username%                                                                                        |
+      | oc:owner-display-name | %displayname%                                                                                     |
```

Comment:
also please verify that this is the expected behaviour of ocis?

Reply:
ok

Reply:
can we do these changes in another PR?